bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,42 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          priorityClassName: "system-node-critical"
+          securityContext:
+            privileged: true
+          serviceAccountName: multus
+          containers:
+            - name: whereabouts
+              image: {{.WhereaboutsImage}}
+              resources:
+                requests:
+                  cpu: "25m"
+                  memory: "25Mi"
+              command:
+              - /ip-reconciler
+              - -kubeconfig=/host/etc/cni/net.d/whereabouts.d/whereabouts.kubeconfig
+              volumeMounts:
+                - name: cni-net-dir
+                  mountPath: /host/etc/cni/net.d
+              securityContext:
+                privileged: true
```

Comment:
The reasons for it are described in https://github.com/openshift/cluster-network-operator/pull/1207/commits/fdf46947684e41f7b3ae97dcda377d2924a8ca8c#r728837192 . 

This pod does **not** require to be privileged - it could regenerate the kubeconfig file instead. 

Your call.